components/script/dom/globalscope.rs

```diff
@@ -2635,6 +2638,7 @@ impl GlobalScope {
         line_number: u32,
         fetch_options: ScriptFetchOptions,
         script_base_url: ServoUrl,
+        can_gc: CanGc,
```

Comment:
We need to `allow` this method to avoid the lint error:
```
error: this function has too many arguments (8/7)
    --> components/script/dom/globalscope.rs:2633:5
     |
2633 | /     pub fn evaluate_script_on_global_with_result(
2634 | |         &self,
2635 | |         code: &SourceCode,
2636 | |         filename: &str,
...    |
2641 | |         can_gc: CanGc,
2642 | |     ) -> bool {
```

Reply:
I'll open a new PR for that?

Reply:
No, it can be addressed in this PR.